LondonBritishLibrary/orient/BLorient692.xml

```diff
@@ -177,143 +177,288 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">202</measure>
-                                    <dimensions type="outer" unit="in">
-                                        <height>14.74</height>
-                                        <width>10</width>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>374</height>
+                                        <width>253</width>
+                                        <depth>115</depth>
                                     </dimensions>
                                 </extent>
-                                <condition key="deficient">Some folia are stained and otherwise damaged.</condition>
+                                <foliation>Foliation numbers in pencil are on upper right edge of each recto folio.</foliation>
+                                <collation>
+<list>
+<item xml:id="q1" n="1">
+<dim unit="leaf">8</dim>
+<locus from="1r" to="8v"></locus>
+</item>
+<item xml:id="q2" n="2">
+<dim unit="leaf">8</dim>
+<locus from="9r" to="16v"></locus>
+</item>
+<item xml:id="q3" n="3">
+<dim unit="leaf">8</dim>
+<locus from="17r" to="24v"></locus>
+</item>
+<item xml:id="q4" n="4">
+<dim unit="leaf">8</dim>
+<locus from="25r" to="32v"></locus>
+</item>
+<item xml:id="q5" n="5">
+<dim unit="leaf">8</dim>
+<locus from="33r" to="40v"></locus>
+</item>
+<item xml:id="q6" n="6">
+<dim unit="leaf">8</dim>
+<locus from="41r" to="48v"></locus>
+</item>
+<item xml:id="q7" n="7">
+<dim unit="leaf">9</dim>
+<locus from="49r" to="56v"></locus>3, stub after 6; 6, stub after 3
```

Comment:
9 leaves would mean, that the quire ranges from f. 49 to f. 57. The indication of single leaves and stubs makes me think, that it is a quinion with two missing folia. That means, it should be 8 leaves here.

Reply:
Thank you Carsten for your revise, I am going through them now.

Reply:
Yes, right. I have written 9 instead of 8. Corrected.